src/app/modules/customer-management/components/customer-info/components/create-customer/create-customer.html

```diff
@@ -1,6 +1,17 @@
 <div class="container">
   <form style="width: 600px;" [formGroup]="customerForm" (ngSubmit)="onSubmit(customerForm.value)">
     <div class="form-group">
+      <div class="form-group">
```

Comment:
This is not going to solve the problem, what we need to do instead is to show the uuid in the table. Please add this column in the customer list, project list, and project type list as well.

Reply:
Got it, Working on that!